packages/ramda-extension/src/rejectEq.js

```diff
@@ -0,0 +1,18 @@
+import { equals, identity, reject, useWith } from 'ramda';
+
+/**
+ * Filters out every value in a list that equals to first argument.
+ *
+ * @func
+ * @category List
+ *
+ * @example
+ *
+ * 		R_.rejectEq('foo', ['foo', 'bar', 'foo', 'bar']); // ['bar', 'bar']
+ *
+ * @sig a -> [b] -> [c]
+ *
+ */
+const rejectEq = useWith(reject, [equals, identity]);
```

Comment:
can be shortened to:
```
useWith(reject, [equals])
```

see https://goo.gl/ppMxgw

Reply:
According to Ramda docs, it is best practice to pass identity.

`If you expect additional arguments that don't need to be transformed, although you can ignore them, it's best to pass an identity function so that the new function reports the correct arity.z

Reply:
ok, lets have it your way